plugins/module_utils/statuscake.py

```diff
@@ -4,7 +4,7 @@
 import sys
 import argparse
 
-logger = logging.getLogger("statuscake")
+logger = logging.getLogger("Statuscake -")
```

Comment:
```suggestion
logger = logging.getLogger("statuscake")
```

Logger names shouldn't contain formatting. [logging.Formatter](https://docs.python.org/3/library/logging.html#logging.Formatter) may be used to achieve desired logging output format.